.github/workflows/ci.yml

```diff
@@ -25,6 +25,13 @@ jobs:
       - name: Check if deployment
         if: ${{ github.ref == 'refs/heads/master' || (contains(github.ref, 'tags') && contains(github.ref, matrix.architecture)) }}
         run: echo "IS_DEPLOYMENT='true'" >> $GITHUB_ENV
+      - name: Deployment Status
+        id: deployment
+        uses: chrnorm/deployment-action@v1.2.0
+        if: ${{ env.IS_DEPLOYMENT == 'true' }}
```

Comment:
@xhudaman can we move this to above the `uses` statement?

Reply:
Sure, I wasn't certain of where to put it